config/routes.rb

```diff
@@ -76,7 +62,7 @@
 
     # Step 3
     get 'signup/educator/apply', action: :educator_sheerid_form, as: :educator_sheerid_form
-    post 'i/sheerid/webhook', action: :sheerid_webhook, as: :sheerid_webhook
+    post 'sheerid/webhook', action: :sheerid_webhook, as: :sheerid_webhook
```

Comment:
the `/i/` in this post path is hard-coded in the sheer id platform and might be difficult to change (no, it is... I check with them).
Is there some nginx magic to handle this?

Reply:
We can add a rewrite I think